tests/TestHelpers/SharingHelper.php

```diff
@@ -158,7 +158,6 @@ public static function createShare(
 			$fd['expireDate'] = $expireDate;
 		}
 		$headers = ['OCS-APIREQUEST' => 'true'];
-
```

Comment:
It would be good for the reviewers (also for your PR) if you can omit these kind of changes

Reply:
I don't think so but it's fine for now.